src/Umbraco.Infrastructure/PropertyEditors/DataValueEditor.cs

```diff
@@ -217,7 +217,7 @@ public virtual object FromEditor(ContentPropertyData editorValue, object current
             var result = TryConvertValueToCrlType(editorValue.Value);
             if (result.Success == false)
             {
-                Current.Logger.Warn<DataValueEditor>("The value {EditorValue} cannot be converted to the type {StorageTypeValue}", editorValue.Value, ValueTypes.ToStorageType(ValueType));
+                Current.Logger.LogWarning("The value {EditorValue} cannot be converted to the type {StorageTypeValue}", editorValue.Value, ValueTypes.ToStorageType(ValueType));
```

Comment:
Should we be injecting the typed logger rather than using `Current.Logger` here?

Reply:
It's a good idea, but I think we need to have this PR merged, and then we could look into it afterwards..